user_manual/ios_app.rst

```diff
@@ -30,29 +30,49 @@ You'll also find links and information at the ownCloud `installation page
 Managing Your Files
 -------------------
 
-All your files on your ownCloud server are listed on the Files page, but are not 
-downloaded to your iPhone or iPad until you tap on them. Downloaded files are 
-marked with a little green arrow. Thumbnails are displayed for both downloaded 
-and not-downloaded files (owncloud 8.0+ only). Click the overflow button at the 
-top right (the three-dot button) to upload files to your server, create a new 
-folder, or sort your files alphabetically or by date.
+All your files on your ownCloud server are listed on the Files page, but are
+not downloaded to your iPhone or iPad until you tap on them. 
+Downloaded files are marked with a little green arrow. 
+Thumbnails are displayed for both downloaded and not-downloaded files (owncloud
+8.0+ only). 
+Click the overflow button at the top right (the three-dot button) to upload
+files to your server, create a new folder, or sort your files alphabetically or
+by date.
 
-The Files button at the bottom takes you back to your main Files page. The 
-Uploads button shows your history of uploads to your ownCloud server. The Shared 
-Links button shows how many files you have shared by link.
+The Files button at the bottom takes you back to your main Files page. 
+The Uploads button shows your history of uploads to your ownCloud server. 
+The Shared Links button shows how many files you have shared by link.
 
 When you're in the Files view, slide any filename to the right to expose your 
-file management options: Trash, Share, and More. The More button has 
-options for Open With, Rename, Move, Available Offline.
+file management options. 
+These are: Trash, Share, and More. 
+The More button has options for Open With, Rename, Move, Available Offline.
 
 .. image:: images/ios-slider.png
    :alt: Slide filename right to expose management options.
    
-The More button has different options for folders: Rename, Move, Download 
-Folder, and Available Offline.   
+The More button has different options for folders. 
+These are: Rename, Move, Download Folder, and Available Offline.   
 
-When you tap any file in the Files view it downloads to your iPhone or iPad, if 
+.. image:: images/ios-file-squirrel.png
+   :alt: Squirrel image in the iOS app
+
+Downloading Files
+~~~~~~~~~~~~~~~~~
+
+.. image:: images/ios-file-downloaded-file.png
+   :alt: A file that has been downloaded successfully in ownCloud
+
+When you tap any file in the Files view it downloads to your iPhone or iPad, if
 it hasn't already been downloaded, and opens a preview. 
+If the file has been successfully downloaded, then it will have a tick next to
+its icon, as in the image above. 
+
+Sharing Files
+~~~~~~~~~~~~~
+
+The little arrow button on the bottom left has options for sharing the file via
```

Comment:
The little arrow button on the bottom left has options for sharing the file via email, instant messaging or other of your apps, saving or copying, and printing.